src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -84,14 +91,39 @@
                              :rows results}))
                 (callback {:error status-text})))}))))})
 
-(react/defc Page
+
+(react/defc SearchSection
   {:render
-   (fn [{:keys [refs]}]
+   (fn [{:keys [props]}]
+     [:div {}
+      [:div {:style {:fontWeight 700 :fontSize "125%"}} "Search Filters:"]
+      [:div {:style {:background (:background-light style/colors) :padding "16px 12px"}}
+       [comps/TextFilter {:ref "text-filter"
+                          :initial-text (:search-text props)
+                          :width "100%" :placeholder "Search"
+                          :on-filter (:on-filter props)}]]])})
+
+
+(def ^:private PERSISTENCE-KEY "library-page")
+(def ^:private VERSION 1)
```

Comment:
No, this is my idea for versioning saved state.  If you change that 1 to a 2, then on the next load, it will ignore the saved thing and go back to the default.